modules/hdf5_handler/bes-testsuite/h5.cf/t_special_char_attr.h5.das.bescmd.baseline

```diff
@@ -1,6 +0,0 @@
-Attributes {
```

Comment:
Actually I check the das output, it should be 
```
Attributes {
    HDF5_GLOBAL {
        String special_char_attr "~`!@#%^&*()-_+={}[]|\\:;\"'?,<>/\013
	";
    }
}
```
Something is not right here. Need to check with James.

Reply:
Yes. There's is something odd going on:
1. I don't understand how the baselines were empty. That's wrong, regardless of other issues.
2. I can see the attribute value is escaped, but it's not behaving as I expect.
"~`!@#%^&*()-_+={}[]|\\:;\"'?,<>/\v\n\r\t" --> "~`!@#%^&*()-_+={}[]|\\\\:;\\\"'?,<>/\\013\n\r\t" but that becomes: 
        String special_char_attr "~`!@#%^&*()-_+={}[]|\\:;\"'?,<>/\013

	";
in the attribute value.

I have to work on this more. 

NB: This uses the ASCII character set, so it will be escaped regardless of the value of the H5.EscapeUTF8Attr key.

Reply:
If you don't have time, you can transfer this ticket to me. Not an excuse. I didn't implement the escape function and didn't pay much attention about the das output.